src/client/dfuse/dfuse_main.c

```diff
@@ -252,69 +252,88 @@ show_help(char *name)
 {
 	printf(
 	    "usage: %s <mountpoint> [pool] [container]\n"
```

Comment:
i believe that the usage should read `usage: %s [OPTIONS] [mountpoint [pool container]]`, agreed?

Reply:
since -m is only deprecated and not removed, there have to be suqre brackets around the mountpoint. and if -m is used (and consequently no mountpoint positional argument), then pool and cont cannot be positional arguments either and must be specified through other means.  and finally, i don't think we want to get either both pool and cont as positional arguments, or none of them

Reply:
`dfuse mount-point pool` is valid usage.

Reply:
I would think
```
usage: %s [OPTIONS] [mountpoint [pool [container]]]
```

Reply:
what is the use case for the usage where a pool arg is given but no cont arg? and where would the container info come from in that case?  i find this extremely confusing, and would disallow confusing combinations unless there's a real use case.

Reply:
The same as for running without a pool option.

Initially running without a pool was going to be the only way to launch dfuse but that was some time ago, with the advent of UNS that's much less commonly recommended or used.

We have discussed removing these two ways of launching before, running without a pool might be needed for the Lustre integration but I'm not sure and it's not a change I'd want to make without signposting the change a long way in advance.

Reply:
You seem to be thinking about different ways to supply options, it *might* be possible to run `dfuse -o container=<cont> <mount_point> <pool>` in that I don't think we explicitly check it and abort but it would be weird is someone wanted to do that and I can't see a valid use-case for it.

What I'm saying is that specifying a pool but not specifying a container is accepted and would work and there's no mixing of how you specify it here, you just specify a pool through one of the three options and don't specify a container anywhere.